spec/presenters/step_by_step_page_presenter_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+require "gds_api/test_helpers/link_checker_api"
+
+RSpec.describe StepByStepPagePresenter do
+  include GdsApi::TestHelpers::LinkCheckerApi
+  include AuthenticationControllerHelpers
+  include GovukContentSchemaTestHelpers
+  include TimeOptionsHelper
+
+  before do
+    allow(Services.publishing_api).to receive(:lookup_content_id)
+    stub_user.name = "Firstname Lastname"
+  end
+
+  describe "#summary_list" do
+    let(:step_nav) { create(:step_by_step_page_with_steps) }
+    let(:default_summary) {
+      {
+        "Status": "Draft",
+        "Last saved": format_full_date_and_time(Time.zone.now),
+        "Created": format_full_date_and_time(Time.zone.now)
+      }
+    }
+
+    subject { described_class.new(step_nav).summary_metadata }
+
+    it "presents a list of key/value metadata" do
+      expect(subject).to eq(default_summary)
+    end
+
+    it "shows who made the most recent change" do
+      step_nav.assigned_to = stub_user.name
```

Comment:
This is the only test that `stub_user` is used. All that's really needed is to assign `step_nav.assigned_to` a string. So rather than including the AuthenticationControllerHelpers and creating a user, you could get away with just doing:

```
step_nav.assigned_to = "Firstname Lastname"
expect(subject["Last saved".to_sym]).to eq("#{format_full_date_and_time(Time.zone.now)} by #{step_nav.assigned_to}")
```